docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -126,13 +137,167 @@ components: sources: kubernetes_logs: {
+				description: "Container image."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["busybox:1.30"]
+					default: null
+				}
+			}
+			"kubernetes.container_name": {
+				description: "Container name."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["coredns"]
+					default: null
+				}
+			}
+			"kubernetes.pod_labels": {
+				description: "Pod labels name."
+				required:    false
+				common:      true
+				type: object: {
+					examples: [{"mylabel": "myvalue"}]
+					options: {}
+				}
+			}
+			"kubernetes.pod_name": {
+				description: "Pod name."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["coredns-qwertyuiop-qwert"]
+					default: null
+				}
+			}
+			"kubernetes.pod_namespace": {
+				description: "Pod namespace."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["kube-system"]
+					default: null
+				}
+			}
+			"kubernetes.pod_node_name": {
+				description: "Pod node name."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["minikube"]
+					default: null
+				}
+			}
+			"kubernetes.pod_uid": {
+				description: "Pod uid."
+				required:    false
+				common:      true
+				type: string: {
+					examples: ["ba46d8c9-9541-4f6b-bbf9-d23b36f2f136"]
+					default: null
+				}
+			}
+			message: {
+				description: "The raw line from the Pod log file."
+				required:    true
+				type: string: examples: ["53.126.150.246 - - [01/Oct/2020:11:25:58 -0400] \"GET /disintermediate HTTP/2.0\" 401 20308"]
+			}
+			source_type: {
+				description: "The name of the source type."
+				required:    true
+				type: string: examples: ["kubernetes_logs"]
+			}
+			stream: {
+				description: "The name of the stream the log line was sumbitted to."
+				required:    true
+				type: string: examples: ["stdout", "stderr"]
+			}
+			timestamp: fields._current_timestamp
+		}
 	}
 
+	examples: [
+		{
+			title: "Sample Output"
+			configuration: {
+				type: "kubernetes_logs"
+			}
+			input: """
+				```text
+				F1015 11:01:46.499073       1 main.go:39] error getting server version: Get \"https://10.96.0.1:443/version?timeout=32s\": dial tcp 10.96.0.1:443: connect: network is unreachable
+				```
+				"""
+			output: log: {
+				"file": "/var/log/pods/kube-system_storage-provisioner_93bde4d0-9731-4785-a80e-cd27ba8ad7c2/storage-provisioner/1.log"
+				"kubernetes": {
+					"container_image": "gcr.io/k8s-minikube/storage-provisioner:v3"
+					"container_name":  "storage-provisioner"
+					"pod_labels": {
+						"addonmanager.kubernetes.io/mode": "Reconcile"
+						"gcp-auth-skip-secret":            "true"
+						"integration-test":                "storage-provisioner"
+					}
+					"pod_name":      "storage-provisioner"
+					"pod_namespace": "kube-system"
+					"pod_node_name": "minikube"
+					"pod_uid":       "93bde4d0-9731-4785-a80e-cd27ba8ad7c2"
+				}
```

Comment:
Should these be nested or should they be top-level keys? Ex: `"kuberenetes.container_image"`?

Reply:
Right, this was just copied from the `stdout` sink output, I'll correct this. Good catch!